src/components/content/survey-table/survey-table.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import './survey-table.scss';
+
+
+class SurveyTable extends React.Component {
+    render() {
+        let classAnimationName = '';
+        if (this.props.opened === 'open') classAnimationName = 'survey-table-down ';
+        if (this.props.opened === 'close') classAnimationName = 'survey-table-up ';
```

Comment:
Also here, opened won't nothing else then `open` or `close`.
As we can see based on it's usage
```jsx
<SurveyTable opened={this.state.surveys[index].opened ? 'open' : 'close'} key={survey.unique}>
```